source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -204,11 +204,30 @@ which collections.
 .. example::
 
    Your app may contain 20 or 30 collections, but you want to minimize the
-   number of queryable fields. You can re-use queryable fields across
-   collections in order to sync objects from every collection. For example,
-   "owner_id" might be a field you want to query in multiple collections, but by
-   having the same field name across collections, only one queryable field is
-   used.
+   number of queryable fields. You can re-use **global queryable fields** 
+   across collections in order to sync objects from every collection. 
+   For example, ``owner_id`` might be a field you want to query in multiple 
+   collections. By making this a **global queryable field** across 
+   collections, only one queryable field is used.
+
+   Alternately, you may have ``owner_id`` in multiple collections, but only
+   need to query on it in one collection. In this case, you might make 
+   ``owner_id`` a **collection queryable field**. This means Sync only has to
+   maintain metadata about this field for one collection, instead of storing
+   metadata for all of the collections where you're not querying on this 
+   field.
+
+   Finally, for Apps where devices want to query one specific facet of the 
+   data, such as ``owner_id == user.id``, you may want to designate the 
+   field an **indexed queryable field**. Indexed queryable fields provide 
+   more efficient storage and performance for Apps where the client only
```

Comment:
2 small notes on this sentence
- IQFs increase performance but don't  result in more efficient storage (in fact it will increase the amount of storage used since we're creating an index)
- The last bit of this sentence reads a bit funny to me. Thoughts on something like this?

`Indexed queryable fields provide more efficient performance for Apps where the client only needs to sync on a small subset of their data - a group of stores or a single user, for example.`